app/views/layouts/application.html.erb

```diff
@@ -98,10 +98,19 @@
             <li><%= link_to "Sign In", new_session_path %></li>
             <li> <%= link_to "Sign Up", new_user_path, id: "btn-sign-up" %></li>
           <% end %>
+
+          <% if user_signed_in? %>
+            <% if current_user.organization_id == nil %>
```

Comment:
It may be a good idea to wrap this in a method considering this is used in multiple place so in `user.rb` you can have:
```
def missing_organization?
  current_user.organization.nil?
end
```